### ISSUE TITLE
Hide in HTML-Tab

### ISSUE THREAD

**Issue by kingmauri** [frctl/fractal#422]
Is there a way to hide components in the html-output-Tab? I want to hide the <hr> tag.
Thx for any suggestions.

{{#each variants}}
    
    
    {{render @root.comprender this merge=true}}
    
    <hr>

{{/each}}

**Comment by kingmauri** [comment]
ok I did this in panel-html.nunj

            {% if not entity.context.hideHtml %}
                <pre>{{ render.entity(entity.render(null, renderEnv, {preview: false, collate: false}) | async(true)) | trim }}</pre>
            {% endif %}

and in the component I added a isHtml in the context.

is there a more elegant way? :)

**Comment by martinsvoboda** [comment]
Yes, you can hide or reorder tabs by configuration. In fractal.js try this:

```
const mandelbrot = require('@frctl/mandelbrot');

const myCustomisedTheme = mandelbrot({
    panels: ["notes", "view", "context", "resources", "info"]
});

fractal.web.theme(myCustomisedTheme);
```
This configuration more Notes tab to the first place and hide HTML tab. More about theme configuring you can find here https://fractal.build/guide/customisation/web-themes
